index.html

```diff
@@ -59,7 +59,7 @@
                 }, {
                     name: "Matthias Kovatsch",
                     company:    "Huawei",
-                    companyURL: "https://www.huawei.com"
+                    companyURL: "https://www.huawei.com/en/?ic_medium=direct&ic_source=surlent"
```

Comment:
I am wondering why we need to add this redirect/updated URLs ?

Reply:
I see but I think it is not useful to have such a change

Reply:
In the end, this specific one is removed since we just put "while at Huawei". Do you think we should not change any redirected URL?

Reply:
In those case I think we should keep the URLs as they were.. even adding "/en" is wrong since people from China will also see the English version even though they should be forwarded to the Chinese one..
The same applies to the Intel link.

Reply:
I think we want the simple URLs.  The more complicated one is more likely to change and then be broken...